Makefile

```diff
@@ -417,7 +417,7 @@ nxOMSGenerateInventoryMof:
 
 nxOMSPlugin:
 	rm -rf output/staging; \
-	VERSION="2.18"; \
+	VERSION="2.19"; \
```

Comment:
hey @chparimi 

lets keep these changes seperate.
this is a big change and I don't want to complicate the changes further. 

it will help me test my changes in isolation in INT, if 2.19 includes only my changes.
you can bump the version to 2.20 (for you your changes)